endoreg_db/models/requirement/requirement_evaluation/requirement_type_parser.py

```
@@ -0,0 +1,102 @@
+    # if data is None:
+    #     raise ValueError(f"No data found for requirement type: {name}")
+    # if name not in data_model_dict:
+    #     raise ValueError(f"Unknown requirement type: {name}")
+
+    # model = data_model_dict[name]
+    # if not isinstance(data, model):
+    #     raise TypeError(
+    #         f"Data type mismatch for {name}: expected {model}, got {type(data)}"
+    #     )
+
+    requirement_type = operator_type_tuple.requirement_type
```

Ruff (0.8.2)

Local variable `requirement_type` is assigned to but never used

Remove assignment to unused variable `requirement_type`

(F841)